index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="home.css">
+    <link rel="stylesheet" href="about.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+    <title>Document</title>
+</head>
+<body>
+    <div id="menu">
+        <button id="menu-close">X</button>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Account</a></li>
+            <li><a href="#">Shop</a></li>
+            <li><a href="#">Return</a></li>
+            <li><a href="#">Track</a></li>
+            <li><a href="#">Cart</a></li>
+        </ul>
+    </div>
+     <div class="header-black">
+            <ul>
+                <li><a href=""><img src="images/facebook-64.png" alt="fb"></a></li>
+                <li><a href=""><img src="images/twitter-64.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">MyPage</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
```

Comment:
- [ ] You have done great so far :+1: :+1:  As part of the requirement for this project you are required to replicate the given design as much as you can. I think the height of this top navigation bar can be reduced to match the given design. looking at the screenshot below, you can see the difference in your implementation. I'll suggest you adjust this. :+1: :100: 

- - Your design:
![Screenshot from 2023-06-28 09-57-57](https://github.com/Lawmsangi/Module-One-Capstone/assets/107632780/7d58ec84-d2ba-47ae-9b30-f15f489ba38b)

- - Given design:
![Screenshot from 2023-06-28 09-57-42](https://github.com/Lawmsangi/Module-One-Capstone/assets/107632780/63b9238b-e9e8-487e-8e55-260ff75833fe)